ios/CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Changelog
+All changes to the software that can be noticed from the users' perspective should have an entry in
+this file. Except very minor things that will not affect functionality, such as log message changes
+and minor GUI adjustments.
+
+### Format
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+Entries should have the imperative form, just like commit messages. Start each entry with words like
+add, fix, increase, force etc.. Not added, fixed, increased, forced etc.
+
+Line wrap the file at 100 chars.                                              That is over here -> |
+
+### Categories each change fall into
+
+* **Added**: for new features.
+* **Changed**: for changes in existing functionality.
+* **Deprecated**: for soon-to-be removed features.
+* **Removed**: for now removed features.
+* **Fixed**: for any bug fixes.
+* **Security**: in case of vulnerabilities.
+
+
+## [Unreleased]
+
+
+## [2020.1] - 2020-04-08
+Initial release. Supports...
```

Comment:
The initial release does not follow the regular changelog categories. Since the first is a bit special. I just tried to outline the functionality that exists.